AdyenDropIn/Presentation/DropInNavigationController.swift

```diff
@@ -49,13 +49,15 @@ internal final class DropInNavigationController: UINavigationController {
     // MARK: - Keyboard
     
     @objc private func keyboardWillChangeFrame(_ notification: NSNotification) {
-        guard let topViewController = topViewController as? WrapperViewController,
-            topViewController.requiresInput,
-            let bounds = notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect
-        else { return }
+        if let bounds = notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect {
```

Comment:
I believe there was cases when updating `keyboardRect ` is not needed like stored card payment.

Reply:
stored payment methods works fine, also i had a problem with AwaitComponent where the preferred height takes into account the keyboard height even though the await view has no fields, and when investigated, found out the thekeyboardRect wasn't reset when leaving the MBWay form, hence the change.